runelite-client/src/main/java/net/runelite/client/plugins/slayer/Task.java

```diff
@@ -124,17 +124,17 @@
 	LAVA_DRAGONS("Lava Dragons", ItemID.LAVA_SCALE, "Lava dragon"),
 	LESSER_DEMONS("Lesser demons", ItemID.LESSER_DEMON_MASK),
 	LIZARDMEN("Lizardmen", ItemID.LIZARDMAN_FANG, "Lizardman"),
-	LIZARDS("Lizards", ItemID.DESERT_LIZARD, "Desert lizard", "Sulphur lizard", "Small lizard", "Lizard"),
+	LIZARDS("Lizards", ItemID.DESERT_LIZARD, 3, ItemID.ICE_COOLER, "Desert lizard", "Sulphur lizard", "Small lizard", "Lizard"),
 	MAGIC_AXES("Magic axes", ItemID.IRON_BATTLEAXE, "Magic axe"),
 	MAMMOTHS("Mammoths", ItemID.ATTACKER_HORN, "Mammoth"),
 	MINIONS_OF_SCABARAS("Minions of scabaras", ItemID.GOLDEN_SCARAB, "Scarab swarm", "Locust rider", "Scarab mage"),
 	MINOTAURS("Minotaurs", ItemID.ENSOULED_MINOTAUR_HEAD),
 	MITHRIL_DRAGONS("Mithril dragons", ItemID.MITHRIL_DRAGON_MASK),
-	MOGRES("Mogres", ItemID.MOGRE),
-	MOLANISKS("Molanisks", ItemID.MOLANISK),
+	MOGRES("Mogres", ItemID.MOGRE, -1, ItemID.FISHING_EXPLOSIVE),
+	MOLANISKS("Molanisks", ItemID.MOLANISK, -1, ItemID.SLAYER_BELL),
 	MONKEYS("Monkeys", ItemID.ENSOULED_MONKEY_HEAD, "Tortured gorilla"),
 	MOSS_GIANTS("Moss giants", ItemID.HILL_GIANT_CLUB),
-	MUTATED_ZYGOMITES("Mutated zygomites", ItemID.MUTATED_ZYGOMITE, 7, ItemID.FUNGICIDE_SPRAY_0, "Zygomite", "Fungi"),
+	MUTATED_ZYGOMITES("Mutated zygomites", ItemID.MUTATED_ZYGOMITE, 7, ItemID.FUNGICIDE_SPRAY_10, "Zygomite", "Fungi"),
```

Comment:
Ideally this should work regardless of the charges of fungicide spray present in your inventory, ala `ItemVariationMapping`. Hence, this line shouldn't need to change

Reply:
true, i think i changed it to test the spray which i had bought